packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -344,9 +380,11 @@ class ReactSixteenAdapter extends EnzymeAdapter {
     };
   }
 
-  createShallowRenderer(/* options */) {
+  createShallowRenderer(options) {
     const adapter = this;
     const renderer = new ShallowRenderer();
+    const { suspenseFallback } = options;
+    const shouldReplaceLazyWithFallback = typeof suspenseFallback === 'undefined' ? true : !!suspenseFallback;
```

Comment:
let's throw here if `typeof suspenseFallback` isn't `undefined` or `boolean`

Reply:
additionally, let's throw if this option is specified at *all* in the other two renderers.